### PR TITLE
Grammar correction in ReactDOMInput.js warning

### PR DESCRIPTION
**Pull request by jakeboone02**
Changed "**a** uncontrolled input" to "**an** uncontrolled input".

### CODE REVIEW

**Comment by jimfb** [comment]
@jakeboone02 You will need to fix the unit tests to match.  See https://github.com/facebook/react/pull/6591/files

**Comment by jakeboone02** [comment]
Thanks, @jimfb.  I didn't see #6591.  Looks like that covers it, so I'll close this one.

**Comment by jimfb** [comment]
@jakeboone02 The original author of that PR hasn't responded in a week.  If you want to fix the tests in your PR, we could merge this one.  Your call.

File: docs/js/react.js [hunk 1]
```diff
@@ -7966,7 +7966,7 @@ var ReactDOMInput = {
       var owner = inst._currentElement._owner;
 
       if ((initialValue || !inst._wrapperState.controlled) && controlled && !didWarnUncontrolledToControlled) {
-        "development" !== 'production' ? warning(false, '%s is changing a uncontrolled input of type %s to be controlled. ' + 'Input elements should not switch from uncontrolled to controlled (or vice versa). ' + 'Decide between using a controlled or uncontrolled input ' + 'element for the lifetime of the component. More info: https://fb.me/react-controlled-components', owner && owner.getName() || 'A component', props.type) : void 0;
+        "development" !== 'production' ? warning(false, '%s is changing an uncontrolled input of type %s to be controlled. ' + 'Input elements should not switch from uncontrolled to controlled (or vice versa). ' + 'Decide between using a controlled or uncontrolled input ' + 'element for the lifetime of the component. More info: https://fb.me/react-controlled-components', owner && owner.getName() || 'A component', props.type) : void 0;
```
Comment (jimfb) [inline docs/js/react.js]: As per note in other PR, this file is auto-generated and should not be changed.  Please update the PR such that this file remains unchanged.

**Comment by jimfb** [comment]
Looks good, just same comment as in the other PR.

**Comment by facebook-github-bot-1** [comment]
@jakeboone02 updated the pull request.

**Comment by jakeboone02** [comment]
OK, working on this.  I'll squash to one commit as well.

**Comment by facebook-github-bot-7** [comment]
@jakeboone02 updated the pull request.

**Comment by jimfb** [comment]
Thanks @jakeboone02!

**Pull request merged**